packages/modelence/src/email/providers/AmazonSesProvider.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  SESClient,
+  SendEmailCommand,
+} from '@aws-sdk/client-ses';
+import { EmailData, EmailProvider } from '../types';
+import { getConfig } from '@/server';
+
+export class AmazonSesProvider implements EmailProvider {
+  private client: SESClient;
+
+  constructor() {
+    const region = String(getConfig('_system.email.ses.region')) || 'us-east-1';
```

Comment:
Can we make this config required? I think defaulting to `us-east-1` can be confusing if someone forgets to specify the region and then is not able to figure out why it doesn't work (if they are running in another region).